src/python/test/test_representations.py

```diff
@@ -87,13 +91,15 @@ def test_distance_transform_consistency():
                                       dict(bandwidth=4.),
                                       dict(rel=1e-3)),
 }
+
+kernel_approx = RBFSampler(gamma=1./2, n_components=100000)
```

Comment:
`.5` is simpler than trying to get a division to do the right thing.
Why define it here instead of where you use it?

Reply:
It mindlessly copied the code from `src/python/example/diagram_vectorizations_distances_kernels.py`... Indeed, 0.5 is better